### PR TITLE
Replaced RFC 5988 with RFC 8288

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Replaced RFC 5988 with RFC 8288
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-385312](https://oktainc.atlassian.net/browse/OKTA-385312)

### CODE REVIEW

File: packages/@okta/vuepress-site/3rd_party_notices/Web_App_Client_JS/index.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ _link-headers_
 
 | VERSION (IF ANY):  |                                                              |
 | ---                | ---                                                          |
-| BRIEF DESCRIPTION: | JavaScript parsing of link headers as described by RFC 5988. |
+| BRIEF DESCRIPTION: | JavaScript parsing of link headers as described by RFC 8288. |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/3rd_party_notices/Web_App_Client_JS/index.md:383]: This should be a link

File: packages/@okta/vuepress-site/docs/reference/api/domains/index.md [hunk 2]
```diff
@@ -617,7 +617,7 @@ The DomainListResponse object defines several properties:
 
 ### Links object
 
-Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations. The Links object is read-only.
+Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations. The Links object is read-only.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/domains/index.md:620]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations. The Links object is read-only.
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -6481,7 +6481,7 @@ Specifies a password for a user. A password value is a **write-only** property.
 
 ### Application Links object
 
-Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.  The Links object is **read-only**.
+Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.  The Links object is **read-only**.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:6484]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.  The Links object is **read-only**.
```

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 4]
```diff
@@ -4055,7 +4055,7 @@ Specifies additional verification data for `token` or `token:hardware` Factors
 
 ### Links object
 
-Specifies link relations (See [Web linking](http://tools.ietf.org/html/rfc5988)) available for the current status of a Factor using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
+Specifies link relations (See [Web linking](http://tools.ietf.org/html/rfc8288)) available for the current status of a Factor using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:4058]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of a Factor using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
```

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 5]
```diff
@@ -4141,7 +4141,7 @@ Push Factors must complete activation on the device by scanning the QR code or v
 
 #### Push Factor Activation Links object
 
-Specifies link relations (See [Web linking](http://tools.ietf.org/html/rfc5988)) available for the Push Factor Activation object using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and operations.
+Specifies link relations (See [Web linking](http://tools.ietf.org/html/rfc8288)) available for the Push Factor Activation object using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and operations.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:4144]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the Push Factor Activation object using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and operations.
```

File: packages/@okta/vuepress-site/docs/reference/api/groups/index.md [hunk 6]
```diff
@@ -1933,7 +1933,7 @@ Profile for a Group that is imported from Active Directory
 
 ### Links object
 
-Specifies link relations. See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the Group using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
+Specifies link relations. See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the Group using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/groups/index.md:1936]:
```suggestion
Specifies link relations. See [Web Linking](http://tools.ietf.org/html/rfc8288) available for the Group using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 7]
```diff
@@ -583,7 +583,7 @@ The Conditions object specifies the conditions that must be met during Policy ev
 
 ### Links object
 
-Specifies Link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current Policy. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
+Specifies Link relations (See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current Policy. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:586]:
```suggestion
Specifies Link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current Policy. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 8]
```diff
@@ -697,7 +697,7 @@ The Conditions object specifies the conditions that must be met during Policy ev
 
 ### Links object
 
-Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current Rule. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
+Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current Rule. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:700]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current Rule. The Links object is used for dynamic discovery of related resources. The Links object is **read-only**.
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 9]
```diff
@@ -4168,7 +4168,7 @@ Specifies the authentication provider that validates the user's password credent
 
 ### Links object
 
-Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current status of a user.  The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations.  The Links object is read-only.
+Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of a user.  The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations.  The Links object is read-only.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:4171]:
```suggestion
Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of a user.  The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations.  The Links object is read-only.
```

**Review by brianduffield-okta — APPROVED** [review]
LGTM - suggested consistency in href formats

**Review by royalchan-okta — APPROVED** [review]

**Pull request merged**